app/services/doc_auth/socure/requests/docv_result_request.rb

```diff
@@ -27,6 +27,14 @@ def handle_http_response(http_response)
           )
         end
 
+        def handle_connection_error(exception:)
+          NewRelic::Agent.notice_error(exception)
+          DocAuth::Socure::Responses::DocvResultResponse.new(
+            http_response: nil,
```

Comment:
as discussed will return DocAuth::Response ... passing nil results in PII parsing exceptions in DocvResultResponse
#<DocAuth::Socure::Responses::DocvResultResponse:0x000000015c292b30 @http_response=nil, @biometric_comparison_required=false, @success=false, @errors={:network=>true}, @exception=#<NoMethodError: undefined method `body' for nil>, @extra={:backtrace=>["/Users/amirreavis-bey/CODE/identity-idp/app/services/doc_auth/socure/responses/docv_result_response.rb:114:in `parsed_response_body'", "/Users/amirreavis-bey/CODE/identity-idp/app/services/doc_auth/socure/responses/docv_result_response.rb:109:in `get_data'"